### PR TITLE
restructure: split client and cli

### PR DESCRIPTION
**Pull request by brech1**
# Description

This PR splits the eigentrust library from the CLI application.

# Related Issues

- #224 

# Changes

- Add `cli` crate.
- Remove environment file usage from the library.
- Update client constructor.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
-	"zk",
-	"client",
+	"eigentrust",
+	"eigentrust-cli",
+	"eigentrust-zk",
 ]
-default-members = ["client"]
+default-members = ["eigentrust-cli"]
```
Comment (lazovicff) [inline Cargo.toml:7]: How do we choose a default?

**Comment by lazovicff** [comment]
`.gitignore` should be updated to ignore params

**Comment by brech1** [comment]
> `.gitignore` should be updated to ignore params

Thanks, I forgot to update the path. Params are ignored with the `/eigentrust/assets/params-*.bin` rule, but I didn't delete them, should I remove them all params?

**Comment by lazovicff** [comment]
> > `.gitignore` should be updated to ignore params
> 
> Thanks, I forgot to update the path. Params are ignored with the `/eigentrust/assets/params-*.bin` rule, but I didn't delete them, should I remove all params?

Yes, we can remove them

**Review by lazovicff — APPROVED** [review]

**Pull request merged**